tests/src/helpers.c

```diff
@@ -282,3 +282,38 @@ void mbedtls_param_failed( const char *failure_condition,
     }
 }
 #endif /* MBEDTLS_CHECK_PARAMS */
+
+#if defined(MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS)
```

Comment:
I'll move it into its own file. That'll make on-target testing for platform providers (like us) easier, since we'd just have to swap out the one file instead of modifying it to add more conditions on its definition.